frontend-web/webclient/app/Accounting/Usage.tsx

```diff
@@ -1413,7 +1414,7 @@ const UsageByUsers: React.FunctionComponent<{loading: boolean, data?: UsagePerUs
                     </thead>
                     <tbody>
                         {data.dataPoints.map(it =>
-                            <tr>
+                            <tr key={it.username + " - " + it.category.name + " (" + getShortProviderTitle(it.category.provider) + ")"}>
```

Comment:
For future reference, this is not readable for anybody other than React internally. Should happen with matching, so shorter is usually better.

For this case, it's probably fine as is.

Reply:
Yes, you'd likely need just something like `{it.username + it.category.name}` to ensure uniqueness in this case.